modules/apps/29-fee/keeper/escrow_test.go

```diff
@@ -55,8 +55,47 @@ func (suite *KeeperTestSuite) TestDistributeFee() {
 				balance = suite.chainA.GetSimApp().BankKeeper.GetBalance(suite.chainA.GetContext(), forward, sdk.DefaultBondDenom)
 				suite.Require().Equal(expectedForwardAccBal, balance)
 
-				// check if the refund acc has been refunded the timeoutFee
-				expectedRefundAccBal := refundAccBal.Add(defaultTimeoutFee[0].Add(defaultTimeoutFee[0]))
+				// check if the refund amount is zero
+				expectedRefundAccBal := refundAccBal
+				balance = suite.chainA.GetSimApp().BankKeeper.GetBalance(suite.chainA.GetContext(), refundAcc, sdk.DefaultBondDenom)
+				suite.Require().Equal(expectedRefundAccBal, balance)
+
+				// check the module acc wallet is now empty
+				balance = suite.chainA.GetSimApp().BankKeeper.GetBalance(suite.chainA.GetContext(), suite.chainA.GetSimApp().IBCFeeKeeper.GetFeeModuleAddress(), sdk.DefaultBondDenom)
+				suite.Require().Equal(sdk.NewCoin(sdk.DefaultBondDenom, sdkmath.NewInt(0)), balance)
+			},
+		},
+		{
+			"success: some refund amount",
```

Comment:
Also here:

```suggestion
			"success: refund timeout_fee - (recv_fee + ack_fee)",
```

Reply:
The success test case above could also be renamed to `"success: no refund"`.

Reply:
done